src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/VedtaksperiodeMigreringsController.kt

```diff
@@ -0,0 +1,102 @@
+package no.nav.tilleggsstonader.sak.vedtak.barnetilsyn
+
+import no.nav.security.token.support.core.api.Unprotected
+import no.nav.tilleggsstonader.kontrakter.felles.mergeSammenhengende
+import no.nav.tilleggsstonader.sak.vedtak.VedtakRepository
+import no.nav.tilleggsstonader.sak.vedtak.VedtakService
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.domain.BeregningsresultatForMåned
+import no.nav.tilleggsstonader.sak.vedtak.domain.GeneriskVedtak
+import no.nav.tilleggsstonader.sak.vedtak.domain.InnvilgelseTilsynBarn
+import no.nav.tilleggsstonader.sak.vedtak.domain.OpphørTilsynBarn
+import no.nav.tilleggsstonader.sak.vedtak.domain.Vedtaksperiode
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping("/admin/vedtak/migrer")
+@Unprotected
+class VedtaksperiodeMigreringsController(
+    val vedtakservice: VedtakService,
+    val vedtakRepository: VedtakRepository,
+) {
+    @GetMapping()
+    fun migrer() {
```

Comment:
Vurder å gjøre alt i en egen tråd, som kjør alt i en egen transaksjon
Så hvis noe går galt, feiler alt. Går alt ok commites alt :D 

```suggestion
    val callId = MDC.get(MDCConstants.MDC_CALL_ID)
        Executors.newVirtualThreadPerTaskExecutor().submit {
            MDC.put(MDCConstants.MDC_CALL_ID, callId)
            try {
                transactionHandler.runInNewTransaction {
                   ...denne jobben
                }
            } catch (e: Exception) {
                secureLogger.error("Feilet jobb", e)
            } finally {
                MDC.remove(MDCConstants.MDC_CALL_ID)
            }
        }
```